websites/M/myCANAL/presence.ts

```diff
@@ -6,62 +6,110 @@ const presence = new Presence({
 		pause: "general.paused",
 		live: "general.live",
 	}),
-	browsingTimestamp = Math.floor(Date.now() / 1000);
-let title;
+	browsingTimestamp = Math.floor(Date.now() / 1000),
+	containsTerm = (term: string) => {
+		if (document.location.pathname.includes(term))
+			return document.location.pathname;
+	};
 
```

Comment:
```ts

enum Assets {
	Logo: "https://i.imgur.com/H3IXznl.png",
	Pause: "(insert link)",
	Play: "(insert link)",
	Live: "(insert link)",
}

```

Reply:
Hello!
Since when Imgur links are preferred rather than Discord asset names?

Reply:
> Hello! Since when Imgur links are preferred rather than Discord asset names?

I'm not sure when exactly it started, but it's been this way for a few months at least.

Reply:
nondependent

Reply:
theusaf is correct by saying this 😊